applications/StatisticsApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -0,0 +1,51 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya (https://github.com/sunethwarna)
+//
+
+
+// System includes
+
+// External includes
+#include <pybind11/pybind11.h>
+#include <pybind11/functional.h>
```

Comment:
what is this used for?

Reply:
This is used to expose a method which returns a std::function to python level